be/src/olap/row_cursor.cpp

```diff
@@ -180,7 +180,7 @@ OLAPStatus RowCursor::allocate_memory_for_string_type(const TabletSchema& schema
     for (auto cid : _schema->column_ids()) {
         fixed_ptr = _fixed_buf + _schema->column_offset(cid);
         FieldType type = schema.column(cid).type();
-        if (type == OLAP_FIELD_TYPE_VARCHAR || type == OLAP_FIELD_TYPE_CHAR) {
+        if (type == OLAP_FIELD_TYPE_VARCHAR || type == OLAP_FIELD_TYPE_CHAR || type == OLAP_FIELD_TYPE_HLL) {
```

Comment:
why not write `variable_ptr = column_schema(cid)->allocate_memory(fixed_ptr + 1, variable_ptr);` directly and without type check.

Reply:
update. I was hurried yesterday ... sorry.